src/app/dim-ui/TileGrid.tsx

```diff
@@ -58,10 +60,9 @@ export function TileGridTile({
     >
       <>
         {icon}
-        <div className={styles.details}>
-          <div className={styles.title}>{title}</div>
-          {children}
-        </div>
+        <div className={clsx(styles.title, corner ? styles.withCorner : undefined)}>{title}</div>
+        {corner || null}
```

Comment:
No need to do `|| null`:

```suggestion
        {corner}
```

Reply:
leftover from being type react.node instead of element. might have been like `""`